docs/storage/multipathing.md

```diff
@@ -0,0 +1,262 @@
+4. Reboot the current XCP-ng host on the affected pool
+5. Do the same for all servers in the pool
+
+#### 2. Prepare the pool
+Make sure that multipathing is enabled on the pool. To do this, go to the advanced configuration of the pool
+
+If this is not the case:
+1. Make sure there are **no VMs running** on an iSCSI and/or HBA SR in the pool
+2. Activate "Enable multipathing for all hosts"
+
+#### 3. Configure the SR
+Proceed with the iSCSI SR configuration as indicated in the [storage documentation](../../storage/#iscsi).
+
+## Fiber Channel (HBA)
+### Requirements
+* Check that the Fiber Channel cards model(s) is supported via the HCL
+* Two different Fiber Channel cards (It can be the same model)
+* Two different SAN switches
+* Multiple targets per LUN on your storage unit
+* Zoning performed
+
+:::warning
+Make sure not to mix Fiber Channel speeds.
+:::
+
+### Target architecture
+#### Target architecture diagram
+```mermaid
+---
+config:
+  look: handDrawn
+  theme: default
+---
+flowchart LR
+  classDef blueNode fill:#A7C8F0,stroke:#2C6693,color:#000000;
+  classDef greenNode fill:#A8E6A2,stroke:#3E8E41,color:#000000;
+  classDef grayNode fill:#D6D6D6,stroke:#8C8C8C,color:#000000;
+
+  subgraph server[XCP-ng host]
+    direction LR
+    subgraph Card2[Fiber Channel card 2]
+        direction RL
+        card2port1[port1]
+    end
+    subgraph Card1[Fiber Channel card 1]
+        direction RL
+        card1port1[port1]
+    end
+  end
+
+  subgraph storage[Storage unit]
+  direction LR
+    subgraph ctrl2[Controller 2]
+        direction RL
+        ctrl2-port1[port1]
+        ctrl2-port2[port2]
+    end
+    subgraph ctrl1[Controller 1]
+        direction RL
+        ctrl1-port1[port1]
+        ctrl1-port2[port2]
+    end
+    lun1@{ shape: lin-cyl, label: "LUN" }
+
+  end
+
+  vlan421[SAN Switch 1]:::blueNode
+  vlan422[SAN Switch 2]:::greenNode
+
+  card1port1<---->vlan421
+  vlan421<---->ctrl1-port1
+  vlan421<---->ctrl2-port1
+
+  card2port1<---->vlan422
+  vlan422<---->ctrl1-port2
+  vlan422<---->ctrl2-port2
+
+  ctrl1 <----> lun1
+  ctrl2 <----> lun1
+
+linkStyle 0 stroke:#4A90E2,stroke-width:2px;
+linkStyle 1 stroke:#4A90E2,stroke-width:2px;
+linkStyle 2 stroke:#4A90E2,stroke-width:2px;
+linkStyle 3 stroke:#5CB85C,stroke-width:2px;
+linkStyle 4 stroke:#5CB85C,stroke-width:2px;
+linkStyle 5 stroke:#5CB85C,stroke-width:2px;
+```
+### Operating procedure
+
+#### 1. Prepare XCP-ng hosts
+1. On one of the host servers, make sure that the multipath.conf configuration includes your storage equipment.
+   
+   This can be found in the file `/etc/multipath.xenserver/multipath.conf`
+2. If your equipment is not present, ask the manufacturer for the multipath configuration for GNU/Linux otherwise move on to the [next step (Prepare the pool)](../../storage/multipathing/#2-prepare-the-pool-1)
+   
+   Add it to the file ```/etc/multipath/conf.d/custom.conf```
```

Comment:
as mentioned in the previous section, an example would be good here to ensure customers get the syntax correct